lms/djangoapps/discussion_api/api.py

```diff
@@ -391,7 +391,10 @@ def get_comment_list(request, thread_id, endorsed, page, page_size, mark_as_read
     # existing comments service interface
     if cc_thread["thread_type"] == "question":
         if endorsed is None:
-            raise ValidationError({"endorsed": ["This field is required for question threads."]})
+            endorsed_responses = cc_thread["endorsed_responses"][response_skip:(response_skip + page_size)]
+            non_endorsed_responses = cc_thread["non_endorsed_responses"]
+            responses = endorsed_responses + non_endorsed_responses
+            resp_total = len(endorsed_responses) + len(non_endorsed_responses)
```

Comment:
To be consistent with line https://github.com/edx/edx-platform/pull/9696/files#diff-225c03df42bb7a286124b09118fed6d8R405 Would it be better to use `cc_thread["non_endorsed_resp_total"]`?